src/style/animatableManager.js

```diff
@@ -0,0 +1,122 @@
+import _ from 'lodash';
+
+
+/** Definitions */
+const definitions = {
+  basicListEntrance: {
```

Comment:
the example screens shouldn't dictate our presets naming, basicList is just an example we have here. 
I would go with more generic names, maybe one that suits the component name, for example, `listItemEnrance`. 

In general, we need to think about naming standards here, because soon enough we'll have all kind of weird, conflicting names. 

maybe the following format: 
[component][entrance/exit/other action][slide/fade/other behaviour]

we can think of something else, as long we have a clear naming convention.

Reply:
Agreed.